path_node:
  ros__parameters:
    path1:
      wp: 5
      wp1_x: 0.80
      wp1_y: 0.0
      wp2_x: 0.0
      wp2_y: -0.80
      wp3_x: -0.80
      wp3_y: 0.0
      wp4_x: 0.0 
      wp4_y: 0.80
      wp5_x: 0.80
      wp5_y: 0.0
    path2:
      wp: 3
      wp1_x: 0.0
      wp1_y: 0.60
      wp2_x: 0.0
      wp2_y: -0.60
      wp3_x: 0.80 
      wp3_y: 0.0
    path3:
      wp: 5
      wp1_x: 0.40
      wp1_y: 0.60
      wp2_x: -0.40
      wp2_y: 0.40
      wp3_x: -0.40
      wp3_y: -0.40 
      wp4_x: 0.40
      wp4_y: -0.60
      wp5_x: 0.80
      wp5_y: 0.0
    path4:
      wp: 12
      wp1_x: 0.40
      wp1_y: 0.20
      wp2_x: 0.40
      wp2_y: 0.60
      wp3_x: 0.0
      wp3_y: 0.40
      wp4_x: -0.40
      wp4_y: 0.60
      wp5_x: -0.40
      wp5_y: 0.20
      wp6_x: -0.80
      wp6_y: 0.0
      wp7_x: -0.40
      wp7_y: -0.20
      wp8_x: -0.40
      wp8_y: -0.60
      wp9_x: 0.0
      wp9_y: -0.40
      wp10_x: 0.40
      wp10_y: -0.60
      wp11_x: 0.40
      wp11_y: -0.20
      wp12_x: 0.80
      wp12_y: 0.0